src/components/accounts/AccountFormAccountId.js

```diff
@@ -133,16 +133,21 @@ class AccountFormAccountId extends Component {
 
     handleCheckAvailability = (accountId, type) => (
         accountId
-            && !(
```

Comment:
nit: I find it pretty hard to read very nested conditional expressions.

Should we convert stuff like this to use multiple statements (e.g. early return pattern)? Something like:

```
if (!accountId) {
   return false;
}
if (this.isImplictAccount()) {
   return type != 'create';
}
if (type == 'create') {
 ....   
}
```

Reply:
Sure thing